### ISSUE TITLE
[🐛 Bug]: Is this selenium issue or code issue?

### ISSUE THREAD

**Issue by ghost** [SeleniumHQ/selenium#12797]
### What happened?

![image](https://github.com/SeleniumHQ/selenium/assets/118899053/3fad1eb8-e320-4e6f-96a7-a407df67ee9e)

Can't fetch element.  

### How can we reproduce the issue?

```shell
iframe_element = find_captcha_iframe(self.driver)
print("The iframe with captcha did not appear")
from ptpython.repl import embed
embed(globals(), locals())

```
```
def find_captcha_iframe(driver):
    try:
        # Find all <div> elements with class "captchaContainer-*" where * is a random string
        div_elements = WebDriverWait(driver, 10).until(EC.presence_of_all_elements_located((By.XPATH, '//div[contains(@class, "captchaContainer-")]')))
        for div_element in div_elements:
            # Check if the class attribute contains "captchaContainer-" followed by a random string
            class_attribute = div_element.get_attribute('class')
            if class_attribute and "captchaContainer-" in class_attribute:
                # Attempt to find a child <div> and an <iframe> within the parent <div>
                child_div = div_element.find_element(By.XPATH, './div')
                iframe = child_div.find_element(By.TAG_NAME, 'iframe')
                if child_div and iframe:
                    return iframe
    except Exception as e:
        print("An error occurred:", e)
        from ptpython.repl import embed
        embed(globals(), locals())
    return None
```
I am using manually to fetch highest level div and go downwards but even highest level div fetching can't be done.
![image](https://github.com/SeleniumHQ/selenium/assets/118899053/b6ca9982-1f52-46ce-8be0-72c69d5ed9c4)
Basically fetching element with aria-label="CAPTCHA" also fails. And that is div for entire captcha box
![image](https://github.com/SeleniumHQ/selenium/assets/118899053/952eebcd-618e-4630-a2aa-e783d6a18016)

### Relevant log output

```shell
Its reproducible easily via discord website.
```

### Operating System

Linux Debian

### Selenium version

selenium selenium==4.11.2

### What are the browser(s) and version(s) where you see this issue?

Chromium 117

### What are the browser driver(s) and version(s) where you see this issue?

Chromedriver 117

### Are you using Selenium Grid?

Not sure
